### PR TITLE
Dinosaurs!

### PR DESCRIPTION
**Pull request by Vonsant**
## About the PR
This PR adds 10 different dinosaur to the game.
All credits for sprites to belay5 (discord)

## Why / Balance
I love dinosaurs. Everyone loves dinosaurs.

## Technical details
10 different dinosaur, dinosaur blood, dinosaur accent, some coctail.
Full list:
1. Compsognathus
2. Velociraptor
3. Triceratops
4. Parasaurolophus
5. Kentrosaurus
6. Dilophosaurus
7. Ankylosaurus
8. Tyrannosaurus
9. Stegosaurus
10. Spinosaurus

## Media
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:
- add: Added dinosaurs!
- add: Added dinosaurs blood
- add: Added dinosaurs accent
- add: Added "Clever Girl" coctail
- add: Added dinosau damage modifier set
- fix: Fixed missing flavor type
-
-->

### CODE REVIEW

**Comment by UbaserB** [comment]
isnt there a mob freeze

**Comment by Vonsant** [comment]
> isnt there a mob freeze

They just got tired and decided to sit down. Yes, that's how it was.

**Comment by metalgearsloth** [comment]
> isnt there a mob freeze

> New pets. While fluff is nice we have enough cargo-obtainable fluff animals and department mascots, to add new animals they must have something interesting like new AI behaviors or spawn in proc gen maps.

Also don't make your CL a shitpost please, it should be technical.

File: Resources/Prototypes/Entities/Mobs/NPCs/dino.yml [hunk 1]
```diff
@@ -0,0 +1,523 @@
...
+    bloodReagent: DinoBlood
+    bloodMaxVolume: 75 #you don't want the map to become pools of blood
+    bloodlossDamage:
+      types:
+        Bloodloss:
+          0.5
+    bloodlossHealDamage:
+      types:
+        Bloodloss:
+          -1
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/Entities/Mobs/NPCs/dino.yml:43]: Your larger / bigger healthpool mobs are going to die of bloodloss a lot easier if you have their blood level this low without changing anything else in bloodstream, especially when they have a higher health pool to take more damage.

Bloodloss damage scales with % blood missing, and it's not that hard to lose 75 blood as a mob that can't tend it's own wounds (ai or player).

having more than or equal to 300 on those isn't a problem (it's also more fun for janitors anyways)

**Comment by K-Dynamic** [comment]
There's something really uncanny about the amount of detail these dinos have

**Comment by Vonsant** [comment]
> Also don't make your CL a shitpost please, it should be technical.

Sorry, fixed.

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/NPCs/dino.yml (hunk 1 above)
Comment (Vonsant) [reply Resources/Prototypes/Entities/Mobs/NPCs/dino.yml:43]: I wanted to make more blood, but I noticed this comment in "argocyte" ( https://github.com/space-wizards/space-station-14/blob/master/Resources/Prototypes/Entities/Mobs/NPCs/argocyte.yml ), even Leviathan has this amount of blood. If you really think it's worth adding, then I'll correct it to more relevant values.

Comment (QuietlyWhisper) [reply Resources/Prototypes/Entities/Mobs/NPCs/dino.yml:43]: I mustve missed those when I changed bloodstream stuff but yes they'd have the same problem

They aren't used in game yet thankfully unless I missed something

Comment (Vonsant) [reply Resources/Prototypes/Entities/Mobs/NPCs/dino.yml]: Done

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]
Cool sprites but i don't see a reason in adding a bunch of dinosaurs if they realistically are never going to be used in game

**Comment by IProduceWidgets** [comment]
Gotta disagree, a Jurassic themed exped seems really funny.

**Review by IProduceWidgets — COMMENTED** [review]
I think they're neat, but they definitely need a place to make sense in. I think you could make an exped work for that.

**Comment by IProduceWidgets** [comment]
I think the PR looks a little odd because you only posted the side sprites. The front/back sprites are also quite good and it doesn't feel any weirder than the Xenos do.

**Comment by Errant-4** [comment]
Just want to point out that we already have Laser Raptors in the codebase which are in a somewhat different style, that would create a theme conflict even if they would probably never show up next to each other (afaik laser raptors aren't even used currently)

image

**Comment by UbaserB** [comment]
laser raptors are found in lottery crates

**Comment by Zaraka** [comment]
where are the feathers? 😎

**Comment by ERORR404V1** [comment]
Why not add another type of enemy on the expeditions....

**Comment by EmoGarbage404** [comment]
I realistically don't see a situation where these are going to be used and I don't think they add a whole lot of value as is. These are additionally pretty wildly out of style to a point where I don't think they should be in. This could be revisited with some different assets if there was ever a proper reason to have dinosaurs in the game.

**Pull request closed without merging**